client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -219,8 +225,8 @@ export function SitesDashboard( {
 								recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
 							} }
 							href={ importSiteUrl }
-							icon="arrow-down"
 						>
+							<UploadIcon icon={ upload } size={ 18 } />
```

Comment:
I set the size to be consistent with the Jetpack icon above but if it looks on a smaller side, we can increase to 20 which also looks good

Reply:
This doesn't have the correct effect on hover:

![CleanShot 2023-12-07 at 04 55 17@2x](https://github.com/Automattic/wp-calypso/assets/36432/7ee4a8e4-f7b5-4a09-802a-29573c0ae412)

Here's what I'd suggest instead:

```diff
diff --git a/client/sites-dashboard/components/sites-dashboard.tsx b/client/sites-dashboard/components/sites-dashboard.tsx
index f2159593868..440457d249f 100644
--- a/client/sites-dashboard/components/sites-dashboard.tsx
+++ b/client/sites-dashboard/components/sites-dashboard.tsx
@@ -145,11 +145,6 @@ const ManageAllDomainsButton = styled( Button )`
 	margin-inline-end: 1rem;
 `;
 
-const UploadIcon = styled( Icon )`
-	margin-right: 8px;
-	vertical-align: bottom;
-`;
-
 const SitesDashboardSitesList = createSitesListComponent();
 
 export function SitesDashboard( {
@@ -225,8 +220,8 @@ export function SitesDashboard( {
 								recordTracksEvent( 'calypso_sites_dashboard_new_site_action_click_import' );
 							} }
 							href={ importSiteUrl }
+							icon={ <Icon class="gridicon" icon={ upload } size={ 18 } /> }
 						>
-							<UploadIcon icon={ upload } size={ 18 } />
 							<span>{ __( 'Import an existing site' ) }</span>
 						</PopoverMenuItem>
 					</SplitButton>
```

Reply:
Hi, thanks for the feedback. I saw the icon from Gridicon but I thought we were moving towards the Gutenberg icons to be consistent. I am happy to add the styles to the Gutenberg icon to look good on hover. 

@javierarce what is your opinion? Should we use the Gutenberg icon instead or it is fine with Gridicon?

Reply:
We should use the Gutenberg one (we are trying to move away from Gridicons as much as possible)